pkg/broker/broker.go

```diff
@@ -105,18 +107,19 @@ func (b *Broker) Provision(request *osb.ProvisionRequest, _ *broker.RequestConte
 		response.OperationKey = &operationKey
 	}
 
-	glog.V(5).Infof("Successfully initiated provisioning %s (%s/%s) in %s", request.InstanceID, request.ServiceID, request.PlanID, namespace)
+	klog.V(4).Infof("broker: provisioned %q in namespace %q", request.InstanceID, namespace)
 	return &response, nil
 }
 
 func (b *Broker) Deprovision(request *osb.DeprovisionRequest, _ *broker.RequestContext) (*broker.DeprovisionResponse, error) {
-	glog.V(5).Infof("Deprovisioning %s (%s/%s)", request.InstanceID, request.ServiceID, request.PlanID)
+	klog.V(4).Infof("broker: deprovisioning request %+v", request)
+
 	b.Lock()
 	defer b.Unlock()
 
 	operationName, err := b.Client.Deprovision(request.InstanceID, request.AcceptsIncomplete)
 	if err != nil {
-		glog.Errorln(err)
+		klog.V(5).Infof("broker: failed to deprovision %q: %v", request.InstanceID, err)
```

Comment:
Same as above: shouldn't errors be in 3 or 4?

Reply:
Fixed with https://github.com/kubernetes-sigs/minibroker/pull/57/commits/05020bc5ad39e314d29af238267f7f19ecb777d1.

I also fixed all the others.